doc/conf.py

```diff
@@ -227,19 +232,27 @@
         # Used in quickstart/osm_quickstart
         'https://www.openstreetmap.org/edit',
 
-        # link exists but link check failes
+        # link exists but link check fails
         # SSL: CERTIFICATE_VERIFY_FAILED
         # Used in sponsors.rst
         'https://www.ntua.gr/en',
         'https://www.ice.ucdavis.edu/',
 
-        # Link with certiciate problems
+        # link exists, with strange redirects, but checks fail
+        'https://webchat.freenode.net/',
```

Comment:
```suggestion
        'https://web.libera.chat/',
```

AFAIK most OSGeo projects have abandoned freenode in favor of libera.

Reply:
The URLs in the conf.py are just URLs to ignore when reporting errors. I can do a global replace for all `webchat.freenode.net` to `web.libera.chat` - there are 22 instances of these. Maybe the URL is also valid so this line in conf.py can then be removed.

Reply:
Updated